docs/watcher.md

```diff
@@ -44,4 +44,4 @@ fusebox({
 });
 ```
 
-*(note: The `ignored` field given to `chokidar` is declared by the `watch` field in FuseBox's configuration.  Do not define the `ignored` field here.)*
+*(note: The `ignored` field given to `chokidar` is declared by the `watch` field in FuseBox's configuration.  Do not define the `ignored` field here.)*
```

Comment:
I think you removed the docs ;-)

Reply:
I removed the docs from the watcher because I moved the config option to the hmr prop in the config.
Should I make a new doc file for the hmr options?

Reply:
sure, would be nice to the docs in place!

Reply:
You can make a new file `hmr.md` whatever works best. We can refactor it later.

Reply:
What does `reloadEntryOnStylesheet` do? - I want to include it in the doc

Reply:
hey, join slack https://slack.fuse-box.org/ it's easier to coordinate there. It's when you'd want to reload an entry point when a css file is changed